steps/steps.py

```diff
@@ -0,0 +1,96 @@
+from behave import *
+from selenium import webdriver
+
+
+import environment
+import time
```

Comment:
I na koniec jeszcze jedna rada o dobrych praktykach. Na to absolutnie nie będzie zwracał uwagi Twój "egzaminator", skoro kod będziesz pisać "na żywo", ale tak na przyszłość: przyjęło się, by importować moduły w takiej kolejności:
- moduły biblioteki standardowej pythona (time)
- biblioteki zewnętrzne (behave, selenium)
- własne moduły (environment)

W każdej z tych grup następująca kolejność: `import module_name` a potem `from module_name import something`, a w każdej z tych podgrup kolejność alfabetyczna :)

W Twoim przypadku powinno wyglądać tak:

    import time
    
    from behave import *
    from selenium import webdriver
    
    import environment

a idealnie powinnaś się pozbyć gwiazdki, czyli wszystkie użyte nazwy zaimportować wprost (mam nadzieję, że niczego nie pominąłem):

    import time
    
    from behave import step, use_step_matcher
    from selenium import webdriver
    
    import environment